nntrainer/app_context.cpp

```diff
@@ -306,6 +307,8 @@ static void add_default_object(AppContext &ac) {
                      LayerType::LAYER_POSITIONAL_ENCODING);
   ac.registerFactory(nntrainer::createLayer<IdentityLayer>, IdentityLayer::type,
                      LayerType::LAYER_IDENTITY);
+  ac.registerFactory(nntrainer::createLayer<Upsample2dLayer>,
```

Comment:
This block is what I mentioned above.
The `registerFactory` is not under the same condition.